src/realm/util/ez_websocket.cpp

```diff
@@ -247,15 +253,16 @@ void EZSocketImpl::initiate_http_tunnel()
     m_proxy_client.emplace(*this, logger());
     auto handler = [this](HTTPResponse response, std::error_code ec) {
         if (ec && ec != util::error::operation_aborted) {
-            m_observer.websocket_http_tunnel_error_handler(ec); // Throws
+            logger().error("Failed to establish HTTP tunnel: %1", ec.message());
+            m_observer.websocket_connect_error_handler(ec); // Throws
             return;
         }
 
         if (response.status != HTTPStatus::Ok) {
             logger().error("Proxy server returned response '%1 %2'", response.status, response.reason); // Throws
             std::error_code ec2 =
                 util::websocket::Error::bad_response_unexpected_status_code; // FIXME: is this the right error?
-            m_observer.websocket_http_tunnel_error_handler(ec2);             // Throws
+            m_observer.websocket_connect_error_handler(ec2);                 // Throws
```

Comment:
Extra indent at end.

Reply:
Our clang-format config is currently set up to keep end of line comments aligned in a block. I don't love this setting, but it is far from my biggest issue.